Sources/GRPC/PlatformSupport.swift

```diff
@@ -155,23 +153,22 @@ public enum PlatformSupport {
     networkPreference: NetworkPreference = .best,
     logger: Logger? = nil
```

Comment:
Why are we defaulting the loggers in this file to `nil` as opposed to the no-op logger?

Reply:
I don't think there's a good reason to use an optional logger here; I just didn't think to change it.